gatsby-node.js

```diff
@@ -0,0 +1,78 @@
+const path = require("path");
+
+/** Create paginated blog post index pages, filtered by topic.
+ *
+ * `topic` should be a BlogPostTopic uid, or null for the case where we list all
+ * posts regardless of topic.
```

Comment:
```suggestion
 * posts regardless of topic.
 *
 * See: https://prismic.io/docs/technologies/paginate-results-gatsby
```

Reply:
For framework-dependent boilerplate heavy stuff like this, I've taken a liking to adding links to the docs -- although I do notice this url doesn't have a version parameter in it...hopefully it doesn't break in the future

Reply:
Good idea, let me add in the suggestion. Even if the URL breaks, we generally should be sticking close to the latest versions of Gatsby and the Gatsby-Prismic plugin, so hopefully our code will still up to date with the documentation.